comp/core/log/params_test.go

```diff
@@ -69,11 +69,11 @@ func TestLogForDaemon_windows(t *testing.T) {
 
 	require.Equal(t, "TEST", params.loggerName)
 	require.Equal(t, "trace", params.logLevelFn(g))
-	require.Equal(t, "", params.logFileFn(g))
-	require.Equal(t, "", params.logSyslogURIFn(g)) // still empty
+	require.Equal(t, "unused", params.logFileFn(g)) // default log file, since log_file isn't set in g
```

Comment:
This makes the test pass, but I wonder if that's what we expected to happen (given that the default log file is called "unused" in this test, it is expected that it ends up being used)?